OsmAnd-java/src/main/java/net/osmand/osm/io/NetworkUtils.java

```diff
@@ -65,9 +56,10 @@ public static String sendGetRequest(String urlText, String userNamePassword, Str
 			return e.getMessage();
 		}
 	}
-	
 	private static final String BOUNDARY = "CowMooCowMooCowCowCow"; //$NON-NLS-1$
-	public static String uploadFile(String urlText, File fileToUpload, String userNamePassword, String formName, boolean gzip, Map<String, String> additionalMapData){
+	public static String uploadFile(String urlText, File fileToUpload, String userNamePassword,
```

Comment:
?? prepareStream

Reply:
We need to take care if input stream is large and will cause out of memory exception inside app! At least we shouldn't make it worse as it seems we did